### DISCUSSION TITLE
Unable to access Individual Seq UI in windows Edge by HTTP on localhost while Chrome does

### DISCUSSION THREAD

**Discussion by golden-aries** [datalust/seq-tickets#1326] [General]
I have not found a way to prevent Edge from sending Upgrade-Insecure-Requests: 1  header.
Seq reacts with redirecting to https, then Edge simply not showing a UI
![datalust-seq-in-edge](https://user-images.githubusercontent.com/88808217/133946877-ac9325f9-cbfc-4ce0-9c62-b5c34fcdf2c3.PNG)

while in chrome it works perfectly fine:

![datalust-seq-in-chrome](https://user-images.githubusercontent.com/88808217/133947645-bb53b78a-10ce-45e4-93ae-f854c95ef6d8.PNG)

**Comment by nblumhardt** [comment]
Hi @golden-aries - thanks for the note.

Edge on Windows works fine for me; I can see there's a `Non-Authoritative-Reason: HSTS` header in the 307 response, which some searching suggests is added by the browser itself when redirecting an HTTP to HTTPS request when a site uses HSTS.

Seq won't be using HSTS locally by default (unless you've enabled it), but my guess is that you've used `https://localhost` from Edge, perhaps while debugging an ASP.NET Core app that uses the self-signed "development" certificate, and that app enabled HSTS for the hostname.

You should be able to undo this by clearing out any saved browsing data for `localhost` from Edge - let me know if this helps!

**Reply by golden-aries** [reply]
Thank you Nicholas, that worked!
I assume your hypothesis was correct.

**Reply by nblumhardt** [reply]
Awesome 👍
